tests/fixtures/optimizer/tpc-h/tpc-h.sql

```diff
@@ -120,68 +120,34 @@ SELECT
   "supplier"."s_address" AS "s_address",
   "supplier"."s_phone" AS "s_phone",
   "supplier"."s_comment" AS "s_comment"
-FROM (
-  SELECT
-    "part"."p_partkey" AS "p_partkey",
-    "part"."p_mfgr" AS "p_mfgr",
-    "part"."p_type" AS "p_type",
-    "part"."p_size" AS "p_size"
-  FROM "part" AS "part"
-  WHERE
-    "part"."p_size" = 15
-    AND "part"."p_type" LIKE '%BRASS'
-) AS "part"
+FROM "part" AS "part"
```

Comment:
none of these sql queries should change, we should isolate all table selects in tpc-h

Reply:
Why?

Reply:
i guess the question is what should be default, it looks like you're changing the default optimized sql to remove isolated selects.